### PR TITLE
UI changes to account for magazine

### PR DESCRIPTION
**Pull request by jsonkao**

### CODE REVIEW

File: src/js/modules/articles/components/ArticleFooter.js [hunk 1]
```diff
@@ -72,20 +73,20 @@ const styles = {
...
-            className={classes.subscribe}
-            href="https://issuu.com/stuyspectator/docs"
-            target="_blank"
-          >
-            {`Volume ${toRoman(article.volume)}, Issue ${article.issue}`}
-            {/* TODO: Lookup table for individual volume/issue links */}
-          </a>
-          .
+          {section.permalink.includes('special-issues') ? (
+            <span><Link to={section.permalink}>{section.name}</Link>, {section.description}</span>
```
Comment (jsonkao) [inline src/js/modules/articles/components/ArticleFooter.js]: This renders, for Undercurrents: "This article appears in print in  <Link to="...">Undercurrents</Link>, The Spectator's Winter 2018 magazine.

File: src/js/modules/articles/components/ArticleHeader.js [hunk 2]
```diff
@@ -94,7 +94,9 @@ const ArticleHeader = ({ classes, article }) => {
         </Link>
         <h1 className={classes.headline}>{article.title}</h1>
         <div className={classes.meta}>
-          <Byline classes={classes} contributors={article.contributors} />
+          {contributors.length > 0 &&
```
Comment (NicholasLYang) [inline src/js/modules/articles/components/ArticleHeader.js]: When does this happen?

File: src/js/modules/core/components/MastheadBar.js [hunk 3]
```diff
@@ -100,7 +100,7 @@ const styles = {
     display: "flex",
     justifyContent: "center",
   },
-  "@media (max-width: 991px)": {
+  "@media (max-width: 1100px)": {
```
Comment (NicholasLYang) [inline src/js/modules/core/components/MastheadBar.js]: Ideally these should be constants

Comment (jsonkao) [reply src/js/modules/core/components/MastheadBar.js]: Not in the scope of this PR

File: src/js/modules/articles/components/ArticleHeader.js (hunk 2 above)
Comment (jsonkao) [reply src/js/modules/articles/components/ArticleHeader.js]: There are no contributors on the recent big sibs Undercurrents article. @DJankauskas I'm not sure if that was a mistake or if the contributor wanted to remain anonymous.

If the latter case let's open an issue to create a `User` to represent anonymity.

**Pull request merged**